tests/playwright-test/golden.spec.ts

```diff
@@ -394,6 +396,190 @@ test('should compare binary', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(0);
 });
 
+test('should throw for invalid maxDiffPixels values', async ({ runInlineTest }) => {
```

Comment:
I'd drop all tests except two smoke tests, one for each option, and leave all the tests for toHaveScreenshot.